pkg/cli/admin/upgrade/rollback/rollback.go

```diff
@@ -33,11 +33,12 @@ func New(f kcmdutil.Factory, streams genericiooptions.IOStreams) *cobra.Command
 		Long: templates.LongDesc(`
 			Rollback the cluster to the previous release.
 
-			Only patch version rollbacks within the same z stream, e.g. 4.y.newer to 4.y.older, are supported.
-			Minor version rollbacks from 4.newer to 4.older are not supported.  Updates to releases other than
-			the most-recent previous release that the cluster was attempting are not supported.  Rolling back
-			re-exposes the cluster to all the bugs which had been fixed from 4.y.older to 4.y.newer.  In most
-			cases, you probably want to understand what's having trouble and roll forward with fixes.
+			Only patch version rollbacks within the same z stream, e.g. 4.y.newer to 4.y.older, are accepted by
```

Comment:
Does it imply that we allow rollback but we don't grantee any bug fixes and technical support?

Reply:
As maintainers of the CVO and this `oc` subcommand, I am trying to explain how the robots work.  How cases opened for Red Hat support get handled, or whether bug-fixes around rollback success get shipped are all policy that is set at higher levels, and the robots I'm in charge of don't know about any of that.  So my goal is to not imply anything about bug fixes and technical support.  Does my suggested wording here accomplish that goal?  Would folks prefer I added an explicit sentence like:

> For information about whether this subcommand is supported by Red Hat, see the associated product docs.

and then docs would grow something like "The `rollback` subcommand is not supported at all yet"?